### PR TITLE
replace pep8 which is deprecated with flake8

### PR DESCRIPTION
**Pull request by eyal-resonai**
Since pep8 is deprecated moving to flake8.
I've added ignore for all the rules which we have violation for and pep8 didn't mind

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,8 +5,8 @@ help:
 	@echo 'Makefile for YABT                                                  '
 	@echo '                                                                   '
 	@echo 'Usage:                                                             '
-	@echo '   make test       Run full test suite with active Python and PEP8 '
-	@echo '   make quicktest  Run test suite with active Python and PEP8      '
+	@echo '   make test       Run full test suite with active Python and FLAKE8 '
```
Comment (itamaro) [inline Makefile:8]: oh no, the indentation on the closing `'` is off now! 😱

File: requirements.txt [hunk 2]
```diff
@@ -6,16 +6,15 @@ munch>=2.1
 networkx>=2.0
 ostrichlib>=0.1
 requests>=2.18.0
-# seems google-cloud-storage needs six without declaring it.
```
Comment (itamaro) [inline requirements.txt:9]: why remove this comment? is it not true?

**Review by itamaro — APPROVED** [review]
how many violations show up without the suppressions?

File: requirements.txt (hunk 2 above)
Comment (eyal-resonai) [reply requirements.txt:9]: that's actually not true I think. 
google-cloud-storage uses it through the api-core which does declare this:
https://github.com/googleapis/python-api-core/blob/master/setup.py#L37

and we did have six installed just with an older package, I actually not sure how package versions is picked up on travis.
I'll leave a comment saying that

File: Makefile (hunk 1 above)
Comment (eyal-resonai) [reply Makefile:8]: the evil find & replace!

**Comment by eyal-resonai** [comment]
> how many violations show up without the suppressions?

we have ~130 errors in 30 files.
I guess we should go over the different errors and see which one we accept and fix and which we leave to ignore. but different PR

**Pull request merged**